src/System.Private.CoreLib/shared/System/BitOps.cs

```diff
@@ -337,20 +341,22 @@ public static int PopCount(ulong value)
                     + Popcnt.PopCount((uint)(value >> 32)));
             }
 
-            //return PopCount((uint)value)
-            //    + PopCount((uint)(value >> 32));
+            //return PopCount((uint)value) + PopCount((uint)(value >> 32));
 
             return SoftwareFallback(value);
 
-            int SoftwareFallback(ulong val) 
+            int SoftwareFallback(ulong v) 
```

Comment:
This software callback is going to work well for 64-bit platforms. The two calls to PopCount(uint) are likely better for 32-bit platforms. Have this under ifdef?

Reply:
OK, sounds good. What labels do I use for 64 - I see the project defines both x64 & arm64

Reply:
Oh, BIT64 and BIT32